systemtests/src/main/java/io/enmasse/systemtest/selenium/page/ConsoleWebPage.java

```diff
@@ -441,6 +441,25 @@ public void openConsolePage() throws Exception {
         }
     }
 
+    public String getDeploymentSnippet() throws InterruptedException {
+        StringBuilder addressSpaceDeployment = new StringBuilder();
+        Thread.sleep(2000);
```

Comment:
What is this for?

Reply:
Added retry loop instead of single sleep to avoid environmet failures